tests/integration/test_vulnerability_detector/test_run_on_start/test_run_on_start.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.vulnerability_detector import callback_detect_vulnerability_scan_started, \
+    callback_detect_vulnerability_scan_finished
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# variables
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_run_on_start.yaml')
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+p = [{'RUN_ON_START': 'yes'}, {'RUN_ON_START': 'no'}]
+m = [{'run_on_start': 'yes'}, {'run_on_start': 'no'}]
+
+# Configuration data
+configurations = load_wazuh_configurations(
+    configurations_path, __name__, params=p, metadata=m)
```

Comment:
This can go on the same line.

Reply:
Done in 98f3a762b6ad89afab9ebb986feb5555545c30fe